test/mason/mason-test/exitCodeTest.comopts

```diff
@@ -0,0 +1 @@
+-M ../../../tools/mason
```

Comment:
I just noticed typo in extension: `comopts` should be `compopts`. However, we are using the directory-wide `COMPOPTS` anyway, so we can remove this file.